From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 15 Jul 2018 23:44:02 +0100
Subject: init: Avoid ABI change for build salt
Forwarded: not-needed

<linux/build-salt.h> indirectly includes <linux/elf.h>, which
apparently adds some type definitions that change the symbol
version for init_uts_ns.

Hide this change from genksyms.

---
--- a/init/version.c
+++ b/init/version.c
@@ -7,7 +7,9 @@
  */
 
 #include <generated/compile.h>
+#ifndef __GENKSYMS__
 #include <linux/build-salt.h>
+#endif
 #include <linux/export.h>
 #include <linux/uts.h>
 #include <linux/utsname.h>
